cmd/kubeadm/app/preflight/checks.go

```diff
@@ -912,25 +912,58 @@ func RunInitNodeChecks(execer utilsexec.Interface, cfg *kubeadmapi.InitConfigura
 		FileAvailableCheck{Path: kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.Etcd, manifestsDir)},
 		HTTPProxyCheck{Proto: "https", Host: cfg.LocalAPIEndpoint.AdvertiseAddress},
 	}
+
+	// File content check for IPV4 and IPV6 are needed if it is -
+	// (dual stack)  `--service-cidr` or `--pod-network-cidr` is set with a IPV4 and IPV6 cidr, `--apiserver-advertise-address` is optional as it can be auto detected.
```

Comment:
```suggestion
	// (dual stack)  `--service-cidr` or `--pod-network-cidr` is set with an IPV4 and IPV6 CIDR, `--apiserver-advertise-address` is optional as it can be auto detected.
```